File: Testes/R/buscadados.r
searchWebData <- function(url, data_type, file_type) {
  response <- readLines(paste0(url, "?data_type=", data_type, "&file_type=", file_type))
  return(response)
}

cat("Digite a URL: ")
url <- readline()
cat("Digite o tipo de dado: ")
data_type <- readline()
cat("Digite o tipo de arquivo: ")
file_type <- readline()

result <- searchWebData(url, data_type, file_type)
cat(result, "\n")

File: Testes/Swift/buscadados.swift
import Foundation

func searchWebData(url: String, data_type: String, file_type: String) -> String? {
    guard let url = URL(string: "\(url)?data_type=\(data_type)&file_type=\(file_type)") else { return nil }
    do {
        let response = try String(contentsOf: url)
        return response
    } catch {
        print(error.localizedDescription)
        return nil
    }
}

print("Digite a URL: ", terminator: "")
if let url = readLine(), !url.isEmpty {
    print("Digite o tipo de dado: ", terminator: "")
    if let data_type = readLine(), !data_type.isEmpty {
        print("Digite o tipo de arquivo: ", terminator: "")
        if let file_type = readLine(), !file_type.isEmpty {
            if let result = searchWebData(url: url, data_type: data_type, file_type: file_type) {
                print(result)
            }
        }
    }
}
